charts/gardener-extension-admission-aws/charts/runtime/templates/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
 {{ include "labels" . | indent 4 }}
 spec:
-  revisionHistoryLimit: 0
+  revisionHistoryLimit: 5
```

Comment:
Why 5 and not 1 (like for all others)? Do we have to expect that many revisions so that 5 is indeed needed?

Reply:
Hmm, no not really. I just set it to 5, because I already had a few occasions where I was still lacking some information with `revisionHistoryLimit=1` because a Deployment was rolled multiple times over an hour or so and when I was trying to investigate the history I was looking for was already purged.

Do you see any problems with setting it to 5? These Deployments are singletons per Garden/Seed cluster, so scalability shouldn't be a concern as for per-controlplane components.

Reply:
No, I was just curious about the reason.

Reply:
Now you know 😉